src/format/opamTypes.mli

```diff
@@ -237,7 +237,7 @@ type 'a inst_action = [
 type 'a concrete_action = [
   | 'a atomic_action
   | `Build of 'a
-  | `Fetch of 'a
+  | `Fetch of 'a * 'a list
```

Comment:
in fact we can even have a `'Fetch of 'a list`. It will just lead to some empty list checks, instead of always merge `'Fetch (p,pl) -> let pl = p::pl in...`

Reply:
that's a good idea, i think it would be easier to use this way

Reply:
updated